Content.Client/Shuttles/UI/NavScreen.xaml.cs

```diff
@@ -70,7 +70,7 @@ public void SetShuttle(EntityUid? shuttle)
         {
             var shipNameParts = metadata.EntityName.Split(' ');
             var designation = shipNameParts[^1];
-            if (designation[2] == '-')
+            if (designation.Length >= 2 && designation[2] == '-')
```

Comment:
```suggestion
            if (designation.Length > 2 && designation[2] == '-')
```
Length must be 3 or greater.  Otherwise this looks good, thank you, and nice catch.